drivers/net/wireless/mediatek/mt76/mt76_connac_mcu.c

```diff
@@ -1529,8 +1529,6 @@ mt76_connac_mcu_build_sku(struct mt76_dev *dev, s8 *sku,
 	int i, offset = sizeof(limits->cck);
 
 	memset(sku, max_txpower, MT_SKU_POWER_LIMIT);
-	if (offchan)
-		return;
```

Comment:
@objelf here we are setting the same power values for off-channels. Is it better to just set default/max values for off-channels/off-band since they will not be actually used?

Reply:
Firmware would manage all channels (e.g hardware scan and network concurrency among multiple clients) so firmware has to know all tx power for all rates on all channels at one time.
That is SDK doing so.